spec/simplecov_merger.rb

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require_relative '../spec/simplecov_helper'
+
+module SimpleCov
+  module ResultMerger
+    def self.merging!
+      SimplecovHelper.configure
+      fix_paths
+      SimpleCov.collate Dir['coverage/**/.resultset.json'] do
+        merge_timeout 365 * 24 * 3600
+      end
+    end
+
+    def self.fix_paths
+      Dir['coverage/**/.resultset.json'].each do |file|
+        content = File.read(file)
+        json = JSON.parse(content)
+        first_key = json.keys.first
+        second_key = json[first_key].keys.first
+        path = json.dig(first_key, second_key).keys.first
+        paths = path.split('identity-idp')
+        first_path = paths.first
+        new_path = Dir.pwd.split('identity-idp').first
+        content.gsub!(first_path, new_path)
+        File.write(file, content)
+      end
```

Comment:
this is really opaque....

1. can we move this file to `lib/simplecov_merger.rb`
2. WDYT of adding a small spec, say `spec/lib/simplecov_merger_spec.rb` with some trivial examples of how this combines files? 😬 
    - I think we could do like a `Dir.mktmpdir do` ... kinda spec and pass that in
    - If a spec is too hectic, maybe an example comment with what some of the JSON is shaped like?

Reply:
yeah, there's lots of clarity/documentation to add, I want to make sure it doesn't break CircleCI

Reply:
This is largely due to the obstacles simplecov has for merging results, which I'm assuming is why codeclimate has its own binary to handle the merging and upload

ex: https://github.com/simplecov-ruby/simplecov/issues/980, https://github.com/simplecov-ruby/simplecov/issues/920

Reply:
I worked on adding specs, but it was more difficult than I hoped. I have the structure of the results files and comments explaining why things are the way they are.

Reply:
Thanks for trying, the comments with examples are super useful